src/code_8001F980.c

```diff
@@ -1209,40 +1207,23 @@ void func_8002276C(void) {
 GLOBAL_ASM("asm/non_matchings/code_8001F980/func_8002276C.s")
 #endif
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-? func_80022B50(void *, s8, s32); // extern
-? func_80022BC4(void *, s8, s32); // extern
-? func_80022DB4(void *, s8, s32); // extern
-? func_800235AC(void *, s8); // extern
-? func_80026A48(s8); // extern
-? func_80030A34(void *); // extern
-
-void func_80022A98(void *arg0, s8 arg1) {
-    s32 temp_v0;
-    s32 temp_v1;
-
-    if ((arg0->unk0 & 0x8000) == 0x8000) {
-        func_80026A48(arg1);
-        func_800235AC(arg0, arg1);
-        temp_v0 = arg0->unkBC;
-        temp_v1 = temp_v0 & 0x4000000;
-        if ((temp_v1 == 0x4000000) || ((temp_v0 & 0x8000000) == 0x8000000)) {
-            if (temp_v1 == 0x4000000) {
-                func_80022B50(arg0, arg1, 0x8000000);
+void func_80022A98(Player* player, s8 arg1) {
+    if ((player->unk_000 & PLAYER_EXISTS) == PLAYER_EXISTS) {
```

Comment:
Does `&= PLAYER_EXISTS) {` match?

Reply:
If it does, and you feel like changing it feel free to change that. If you don't feel like it. Then I'll merge. lgtm.

Reply:
I tried that before, unfortunately it doesn't seem to match.